### PR TITLE
[4.0] change %s to %d

### PR DESCRIPTION
**Pull request by brianteeman**
IF you're going to do it then do it right ;) #34844

@HLeithner @wilsonge

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f79b3ce178554a2a77cf6113d7e8d28d6e9f79e2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34845">issues.joomla.org/tracker/joomla-cms/34845</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on f79b3ce178554a2a77cf6113d7e8d28d6e9f79e2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34845">issues.joomla.org/tracker/joomla-cms/34845</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34845">issues.joomla.org/tracker/joomla-cms/34845</a>.</sub>

**Comment by Kostelano** [comment]
Apparently, you also need it here: https://github.com/joomla/joomla-cms/blob/4.0-dev/installation/language/en-GB/joomla.ini#L295 (+ similarly for en-US).

@brianteeman Can you explain how such a replacement is explained? Thanks.

**Comment by brianteeman** [comment]
@Kostelano see https://github.com/joomla/joomla-cms/issues/34349#issuecomment-852539471

**Comment by HLeithner** [comment]
> Apparently, you also need it here: https://github.com/joomla/joomla-cms/blob/4.0-dev/installation/language/en-GB/joomla.ini#L295 (+ similarly for en-US).

Yes every %s what is a number should be changed to %d but I only fixed the reported string by the translator. If someone is motivated to check all language file for wrong translations it would be really appreciated.

**Comment by brianteeman** [comment]
> . If someone is motivated to check all language file for wrong translations it would be really appreciated.

I was and that is what this PR is. I just forgot to check the installation folder which I will do now

**Comment by HLeithner** [comment]
> > . If someone is motivated to check all language file for wrong translations it would be really appreciated.
> 
> I was and that is what this PR is. I just forgot to check the installation folder which I will do now

Oh I mixed up the PR thought that's my second pr, good then I hope you already found everything.

File: installation/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -196,7 +196,7 @@ INSTL_DISPLAY_ERRORS="Display Errors"
 INSTL_ERROR="Error"
 INSTL_ERROR_DB="Some errors occurred while populating the database: %s."
 INSTL_ERROR_INITIALISE_SCHEMA="Can't initialise database schema."
-INSTL_EXTENSION_AVAILABLE="%s Available"
+INSTL_EXTENSION_AVAILABLE="%d Available"
```
Comment (Kostelano) [inline installation/language/en-GB/joomla.ini]: This is not a numeric value. This is a string of a parameter of the type of support Zlib, etc.

Comment (brianteeman) [reply installation/language/en-GB/joomla.ini]: oops

**Comment by brianteeman** [comment]
> > > . If someone is motivated to check all language file for wrong translations it would be really appreciated.
> > 
> > 
> > I was and that is what this PR is. I just forgot to check the installation folder which I will do now
> 
> Oh I mixed up the PR thought that's my second pr, good then I hope you already found everything.

No problem. Can't guarantee I found them all but definitely found the majority

**Comment by bembelimen** [comment]
@brianteeman if you could fix the conflicts, we can merge it.

**Comment by brianteeman** [comment]
conflicts resolved

**Comment by HLeithner** [comment]
Sry didn't See it earlier, can you please remove the US translation changes. This would make problems with the deployment script

**Comment by brianteeman** [comment]
@HLeithner done

**Comment by HLeithner** [comment]
thx

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**